src/main/java/org/folio/processing/events/services/publisher/KafkaEventPublisher.java

```diff
@@ -129,6 +130,11 @@ private void checkAndAddHeaders(String recordId, String chunkId, String jobExecu
     } else {
       headers.add(KafkaHeader.header(CHUNK_ID_HEADER, chunkId));
     }
+    if (StringUtils.isBlank(jobExecutionId)) {
+      LOGGER.warn("checkAndAddHeaders:: jobExecutionId is empty");
```

Comment:
@JavokhirAbdullayev  Can we log here smth? Maybe recordId/chunkId? It would be great for the future issues-resolving via logs.